pubspec.yaml

```diff
@@ -5,7 +5,7 @@ description: The 4th generation of ÉTSMobile, the main gateway between the Éco
 # pub.dev using `pub publish`. This is preferred for private packages.
 publish_to: 'none' # Remove this line if you wish to publish to pub.dev
 
-version: 4.9.11+1
+version: 4.10.0+1
```

Comment:
I think it would be better to bump up the patch number instead of the minor version number (4.9.12 instead of 4.10.0) since this is more of an improvement than a new feature, but that's no big deal (and I might be mistaken) 🙂

Reply:
C'est moi qui a dit de faire ça en fait oupsiiiie! C'est vrai que c'est plus un improvement qu'un feature j'avoue :smile: T'en penses quoi Antoine?

Reply:
Ah bah suuuuper :joy:
En vrai je sais pas trop non plus haha, je pense qu'on s'en fiche un peu dans ce cas (en plus si le deploy des home screen widgets marche on sera rendus à la 4.10 anyway 👀 )

Reply:
Je pense que comme cam a merge le rating c'est déjà à 4.10, donc j'pense que tu devrais mettre 4.10.1 sur cette PR là @AntoineLemelin vu que c'est plus un improvement d'un feature déjà existant! Milles excuse pour la confusion ;)

Reply:
Je pense que ça fait du sens il n'y a pas tant un gros feature.